sys/fatfs_diskio/native/Makefile

```diff
@@ -0,0 +1,9 @@
+MODULE = fatfs_diskio_native
+
+INCLUDES += -I$(RIOTBASE)/sys/include/fatfs_diskio
+
+FATFS_DISKIO_NATIVE_DEFAULT_FILE ?= \"../../riot_fatfs_disk.img\"
```

Comment:
This should also be added to `tests/pkg_fatfs/Makefile`, further I think its not wise to look for a file 2 dirs up by default. This clearly depends where you build and exec the binary. Hence, should be just `riot_fatfs_disk.img`.